internal/usage/usage.go

```diff
@@ -337,4 +337,6 @@ const (
 	DisableSchemaAdvisor                      = "Flag that disables the Schema Advisor project setting."
 	EnableRealtimePerformancePanel            = "Flag that enables the Real Time Performance Panel project setting."
 	DisableRealtimePerformancePanel           = "Flag that disables the Real Time Performance Panel project setting."
+	CloudProvider                             = "Name of the provider of the cloud service where Atlas can access the S3 bucket. Atlas only supports AWS."
+	IamRoleID                                 = "Unique identifier of the role that Atlas can use to access the bucket."
```

Comment:
```suggestion
	ExportBucketIAMRoleID                                 = "Unique identifier of the role that Atlas can use to access the bucket."
```
Given the language is export bucket specific is better to avoid someone accidentally reusing it